### DISCUSSION TITLE
[Question] How to delete characters in-game?

### DISCUSSION THREAD

**Discussion by LucasGGamerM** [cxong/cdogs-sdl#868] [Q&A] [answered]
I have accidentally made a duplicate character. How can I delete it without going into the game files and manually removing it?

**Comment by cxong** [comment]
Do you mean in the game editor? You can click Remove in the character editor
https://raw.githubusercontent.com/cxong/cdogs-sdl/gh-pages/_posts/cdogs_nuklear_char_editor.gif

Please note that characters are referenced by index in the game, so it may mess up the missions if you delete characters in the middle. It's safer to ignore - there is no limit to the number of characters. There is a fix in the backlog #396

**Comment by LucasGGamerM** [comment]
I mean in the saved characters you use when you play the game, the characters you customize before entering a quick play session

**Comment by cxong** [comment, marked as answer]
you can't delete those at the moment
